pkg/kubernetes/run.go

```diff
@@ -360,7 +361,11 @@ func writeYamlFile(app runfileconfig.App, svc serviceConfig, dep deploymentConfi
 }
 
 func deployYamlToK8s(yamlToDeployPath string) error {
-	_, err := utils.RunCmdAndWait("kubectl", "apply", "-f", yamlToDeployPath)
+	_, err := os.Stat(yamlToDeployPath)
+	if os.IsNotExist(err) {
+		return fmt.Errorf("error given file %q does not exist", yamlToDeployPath)
```

Comment:
```suggestion
		return fmt.Errorf("error in deploying file %q to k8s environment: %w", yamlToDeployPath, err)
```

Reply:
this is specifically a does not exist error ... So I think should be fine.